sys/drv/plic.c

```diff
@@ -0,0 +1,190 @@
+#define KL_LOG KL_DEV
+#include <sys/bus.h>
+#include <sys/devclass.h>
+#include <sys/errno.h>
+#include <sys/fdt.h>
+#include <sys/interrupt.h>
+#include <sys/klog.h>
+#include <sys/libkern.h>
+
+/* PLIC memory map. */
```

Comment:
Could you put a link to the specification here?

Reply:
Done.